pitch.md

```diff
@@ -1,44 +1,160 @@
+up new repeated events that were already set up for other times. 
+
+While a UI would probably be the final product of this project, it would be 
+nicely represented with a backend language. An expressive language allows
+for small changes to have large effects. If a user's calendar is always 
+recorded in the language instead of chunked time-slots, making a small change
+to the initial design of the calendar would just require a small change in the
+language, not having to go event by event to make that small change. 
 
 ### What's the need?
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+The need that is met by my idea is the need for complex calendar design. Current
+calendar software makes it too hard and work intensive to automate large 
+and complex calendars for most users. Their current experience depends on the 
+calendar software they use but most have options to repeat events over a span
+of time. If they want multiple spans of time, then they have to create multiple
+events. If multiple events have the same span of time, there aren't easy ways
+to group those together. So changing all time spans that are the same at the same
+time is impossible, they all have to be changed separately. 
+
+Their experience using my project would improve the expressiveness of what
+calendars are able to understand. They would be able to group events together
+under the same span of time, make them easily repeat, make it easy for certain 
+events to have exceptions at certain times and much more. It just allows users
+to create calendars the same way they have a calendar arranged in their head, by
+groups. 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+DSLs aim to improve expressiveness over a domain. So since I feel like the domain
+of calendar creation and editing has been stagnant for quite a while, a DSL that
+aims to improve the ease of making and editing one seems to fit quite well. 
+
+Most of the time spent on a calendar is in it's creation. Basically when you are
+adding a set of events that all go together. Imagine you just set up your next 
+semester's schedule for classes, they will all have the same time span of existance,
+so making a new time-section of the calendar with each class being a different 
+repeated events makes intuitive sense. After filling in the dates of breaks and each
+classes time slot, not much needs to be changed until you get your homework schedule.
+
+Instead of having to redo these restrictions of the school year calendar, you can just
+add a new section to your current school year calendar that is just homework. It doesn't
+require repetition or much manual input. By having this calendar designer be a DSL,
+it becomes much easier to make drastic changes with less effort. 
```

Comment:
I wonder how this plays out with exporting this calendar to something like Google Calendar.  I imagine it could be tricky to update your script, generate a new calendar, and then have it replace the old one.